### PR TITLE
docs: Mention about run-tests in contributing.md

### LINKED ISSUE #1193: [Docs] Mention about `make run-tests` in contributing.md
**Current State:**

We now have ability to test some of the CI checks on local machine using `make run-tests`, there is no mention of this command in our contributing file & respective docs.

**Desired State:**
Mention about `make run-tests` in contributing.md

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by Adeola-Adesoba**
**Description**

This PR fixes #1193 

**Notes for Reviewers**
I updated the contributing.md doc to include 'make run-tests' under subsection Tests

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -94,6 +94,14 @@ Any time changes are made to the GO code, you will have to stop the server and r
 Once the Meshery server is up and running, you should be able to access Meshery on your `localhost` on port `9081` at `http://localhost:9081`. One thing to note, you might NOT see the [Meshery UI](#contributing-ui) until the UI code is built as well.
 After running Meshery server, you will need to select your **Cloud Provider** by navigating to `localhost:9081`. Only then you will be able to use the Meshery UI on port `3000`.
 
+### Tests
+We now have ability to test some of the CI checks on your local machine using `make run-tests`.
+ 
+To test some of the CI checks on your local machine:
```
Comment (kushthedude) [inline CONTRIBUTING.md:100]:
```suggestion
To test code changes on your local machine, run the following command :
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -94,6 +94,14 @@ Any time changes are made to the GO code, you will have to stop the server and r
 Once the Meshery server is up and running, you should be able to access Meshery on your `localhost` on port `9081` at `http://localhost:9081`. One thing to note, you might NOT see the [Meshery UI](#contributing-ui) until the UI code is built as well.
 After running Meshery server, you will need to select your **Cloud Provider** by navigating to `localhost:9081`. Only then you will be able to use the Meshery UI on port `3000`.
 
+### Tests
+We now have ability to test some of the CI checks on your local machine using `make run-tests`.
```
Comment (kushthedude) [inline CONTRIBUTING.md:98]:
```suggestion
Users can now test their code on their local machine against the CI checks implemented using `make run-tests`.
```

Comment (Adeola-Adesoba) [reply CONTRIBUTING.md:98]: ok, making the changes now

**Comment by kushthedude** [comment]
Hi @Adeola-Adesoba Can you sign-off your latest commit?

**Comment by Adeola-Adesoba** [comment]
> Hi @Adeola-Adesoba Can you sign-off your latest commit?

Can you post the link here? I can't find it...

**Comment by Adeola-Adesoba** [comment]
> > Hi @Adeola-Adesoba Can you sign-off your latest commit?
> 
> Can you post the link here? I can't find it...

Just did, sign off amended. Thanks.

**Comment by kushthedude** [comment]
Great work @Adeola-Adesoba! Just a suggestion, please name your PR related to the issue you are solving.

**Comment by Adeola-Adesoba** [comment]
> Great work @Adeola-Adesoba! Just a suggestion, please name your PR related to the issue you are solving.

Sure. Thanks.

**Pull request merged**